Formula/n98-magerun.rb

```diff
@@ -0,0 +1,15 @@
+class N98Magerun < Formula
+  desc "The swiss army knife for Magento developers, sysadmins and devops."
+  homepage "http://magerun.net/"
+  url "https://files.magerun.net/n98-magerun.phar"
+  version "1.0.0"
```

Comment:
OK actually I wrote `1.0.0` because I did not really understand what the formula version number reference for. I will write `1.97.22`